cmd/util/util.go

```diff
@@ -0,0 +1,26 @@
+package util
```

Comment:
I removed this file.

Instead of IsEmpty and IsNotEmpty, I switched back to len(var) == 0 and len(var) > 0, this conforms with the rest of the CLI code.